resizer/package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
It seems that other samples don't include a `package.json` and just build by running `tsc` directly. I think it would be good to talk to Anthony about if he's planning on changing this.